src/main/java/com/grepp/carrierroute/booking/domain/HotelBooking.java

```diff
@@ -0,0 +1,67 @@
+package com.grepp.carrierroute.booking.domain;
+
+import com.grepp.carrierroute.hotel.domain.HotelRoom;
+import com.grepp.carrierroute.hotel.exception.ErrorMessage;
+import com.grepp.carrierroute.booking.exception.InvalidHotelBookingParameterException;
+import com.grepp.carrierroute.user.domain.User;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "hotel_booking")
+@Getter
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
접근 제한자 설정해서 무분별한 객체 생성도 막을 수 있을 것 같네용